server/models/pattern/patterns/patterns.go

```diff
@@ -45,34 +48,68 @@ func ProcessOAM(kconfigs []string, oamComps []string, oamConfig string, isDel bo
 		wg.Add(1)
 		go func(kcli *kubernetes.Client) {
 			defer wg.Done()
+			id, _ := uuid.NewV4()
 			for _, comp := range comps {
+				var req meshes.EventsResponse
 				if comp.Spec.Type == "Application" {
 					if err := application.Deploy(kcli, comp, config, isDel); err != nil {
+						req = meshes.EventsResponse{
```

Comment:
Also, please send component that generated this `event` and `name` of the component, `Component: core/server`, `ComponentName: meshery` as we do for adapters `COmponent: adapter` `COmponentName: Istio`

Reply:
Resolve the conversation after review.